lessons/6_objects_intro/homework.md

```diff
@@ -0,0 +1,23 @@
+# Lesson 6 Homework
+
+Start working on your little online store prototype and practice objects and arrays
+
+> Important: do not forget to do your homework on a separate branch with meaningful name
+
```

Comment:
Spec?

Reply:
Just copy pasted structure from lesson 5. How can I improve it?

Reply:
Ah, sorry. Just put ## Spec before explanation what should have been done. The same as in classwork

Reply:
done